easybuild/easyconfigs/d/DBus/DBus-1.10.12-intel-2016b.eb

```diff
@@ -16,7 +16,8 @@ source_urls = ['http://dbus.freedesktop.org/releases/dbus']
 sources = [SOURCELOWER_TAR_GZ]
 
 sanity_check_paths = {
-    'files': ['bin/dbus-%s' % x for x in ['cleanup-sockets', 'daemon', 'launch', 'monitor', 'run-session', 'send', 'uuidgen']] +
+    'files': ['bin/dbus-%s' % x for x in ['cleanup-sockets', 'daemon', 'launch', 'monitor', 'run-session', 'send',
+              'uuidgen']] +
```

Comment:
@migueldiascosta please align this with `'cleanup-sockets'`, and maybe also wrap `'send'` to this line to make `'uuidgen'` less lonely ;)